packages/input-checkbox/src/styledComponents/Checkbox.js

```diff
@@ -19,11 +20,88 @@ const baseColors = props => {
   `;
 };
 
-const hoverColors = props => {
+const checkboxStyles = props =>
+  !props.switch &&
+  css`
+    width: ${checkboxSize};
+    height: ${checkboxSize};
+    font-size: 8px;
+    border-radius: 4px;
+
+    .checkmark {
+      visibility: ${props => (props.checked ? "visible" : "hidden")};
+      display: flex;
+      justify-content: center;
+      width: 100%;
+    }
+    ${checkboxColors};
+  `;
+
+const switchBackground = props => {
```

Comment:
Since this function only injects the background-color, it should only return the color. 
And we can use it like this
```
const switchBackground = props => {
  if (props.disabled) {
    return colors.GREY_32;
  }
  // and so on
};

const switchStyles = props => {
  // ...
  background: ${switchBackground}
}
```